server/liveblog/themes/themes.py

```diff
@@ -70,9 +75,27 @@ def get_local_themes_packages(self):
             for file in glob.glob(embed_folder + '/**/theme.json'):
                 yield json.loads(open(file).read())
 
+    def get_themes(self):
+        res = get_resource_service('themes').get(req=None, lookup={})
```

Comment:
you already are in the theme service, use `self` to retrieve it instead of `get_resource_service('themes')`